staging/src/k8s.io/component-base/cli/flags/example/main.go

```diff
@@ -0,0 +1,26 @@
+package example
+
+// Parse parses the command line, potentially calls load and then returns a configuration.
+func Parse(cmdLine []string, load func(fname string, dest *ComponentConfigControllerManager)) (*ComponentConfigControllerManager, error) {
```

Comment:
Ah, ok this is just an example of how one could use the framework? For a second I got confused on why this was coupling parse/load/apply together.